src/thunderbots/software/network_input/backend.cpp

```diff
@@ -15,7 +15,7 @@ Backend::Backend()
           Util::Constants::ROBOT_DEBOUNCE_DURATION_MILLISECONDS)),
       enemy_team_state(Duration::fromMilliseconds(
           Util::Constants::ROBOT_DEBOUNCE_DURATION_MILLISECONDS)),
-      ball_filter(),
+      ball_filter(4, 10),
```

Comment:
What are these numbers? Can they be declared somewhere?

Reply:
The min and max sizes of the buffer. I've made these static member variables.